javascript/javascript3/week3/makeArt.js

```diff
@@ -0,0 +1,24 @@
+let canvas = document.getElementById("myCanvas");
+let ctx = canvas.getContext("2d");
+//making canvas size same as window
+let window_width = window.innerWidth;
+let window_height = window.innerHeight;
+canvas.width = window_width;
+canvas.height = window_height;
+//let width = canvas.width;
+//let height = canvas.height;
+setInterval(() => {
+    function randomCircle() {
+        let randomX = Math.random() * canvas.width;
```

Comment:
simplify by creating the function getRandomInt(number). Also don't forget to use Math.floor()